DigitalLearningSolutions.Web/Scripts/spec/learningMenu/fullscreen.spec.ts

```diff
@@ -125,8 +123,8 @@ describe('enterFullscreen', () => {
     enterFullscreen();
 
     // Then
-    const iframe = document.getElementById('content-viewer_iframe');
-    expect(iframe!.classList).toContain('fullscreen');
+    const wrapper = document.getElementById('content-viewer_wrapper');

```

Comment:
Minor but I think `content-viewer_iframe-wrapper` is a more descriptive name 🙂

Reply:
Would you like me to rename the class as well?

Reply:
Yes please 🙂